docker/mongodb.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+  mongodb:
+    image: mongo:4.4
+    command: mongod --wiredTigerCacheSizeGB 1.5
+    ports:
+      - "127.0.0.1:${MONGO_EXPOSE_PORT:-27017}:27017"
```

Comment:
sorry, I didn't spot it before, but we need a volume here ! If not mongo data won't be persistent…

Reply:
```suggestion
services:
  mongodb:
    image: mongo:4.4
    command: mongod --wiredTigerCacheSizeGB 1.5
    ports:
      - "127.0.0.1:${MONGO_EXPOSE_PORT:-27017}:27017"
    volumes:
      - mongodb_data:/data/db
      
volumes:
  mongodb_data:
```

Reply:
Indeed!